Projects/ChangeTier/Sources/Views/ChangeTierLandingScreen.swift

```diff
@@ -0,0 +1,245 @@
+                            .hFieldTrailingView {
+                                Image(uiImage: hCoreUIAssets.lock.image)
+                                    .foregroundColor(hTextColor.Opaque.secondary)
+                            }
+                        }
+                    } else {
+                        DropdownView(value: vm.selectedTier?.name ?? "", placeHolder: L10n.tierFlowCoveragePlaceholder)
+                        {
+                            selectTierNavigationVm.isEditTierPresented = true
+                        }
+                    }
+
+                    DropdownView(
+                        value: vm.selectedDeductible?.deductibleAmount?.formattedAmount ?? "",
+                        placeHolder: L10n.tierFlowDeductiblePlaceholder
+                    ) {
+                        selectTierNavigationVm.isEditDeductiblePresented = true
+                    }
+                    .disabled(vm.selectedTier == nil)
+                }
+                .hFieldSize(.small)
+                .hWithTransparentColor
+                .hWithoutHorizontalPadding
+
+                hRow {
+                    HStack(alignment: .top) {
+                        hText(L10n.tierFlowTotal)
+                        Spacer()
+                        VStack(alignment: .trailing, spacing: 0) {
+                            if let newPremium = vm.newPremium {
+                                hText(newPremium.formattedAmountPerMonth)
+                            } else {
+                                hText(vm.currentPremium?.formattedAmountPerMonth ?? "")
+                            }
+
+                            if vm.newPremium != vm.currentPremium {
+                                hText(
+                                    L10n.tierFlowPreviousPrice(vm.currentPremium?.formattedAmountPerMonth ?? ""),
+                                    style: .label
+                                )
+                                .foregroundColor(hTextColor.Opaque.secondary)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    private var buttons: some View {
+        hSection {
+            VStack(spacing: .padding8) {
+                hButton.LargeButton(type: .ghost) {
+                    selectTierNavigationVm.isCompareTiersPresented = true
+                } content: {
+                    hText(L10n.tierFlowCompareButton, style: .body1)
+                }
+                hButton.LargeButton(type: .primary) {
+                    /** TODO: ADD ACTION **/
+                } content: {
+                    hText(L10n.generalContinueButton)
+                }
+                .disabled(!vm.isValid)
+            }
+        }
+        .sectionContainerStyle(.transparent)
+    }
+}
+
+public class SelectTierViewModel: ObservableObject {
+    @Inject var service: SelectTierClient
+    var displayName: String?
```

Comment:
Needs to be @Published since its used for the UI
Same for the exposureName and others used in the UI